src/Kros.AspNetCore/Extensions/HttpResponseExtensions.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Primitives;
+using Microsoft.Net.Http.Headers;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Kros.AspNetCore.Extensions
+{
+    /// <summary>
+    /// Extensions for <see cref="HttpResponse"/>.
+    /// </summary>
+    public static class HttpResponseExtensions
+    {
+        private static readonly HashSet<string> _preservedHeaders = new HashSet<string>{
+            HeaderNames.AccessControlAllowCredentials, HeaderNames.AccessControlAllowHeaders,
+            HeaderNames.AccessControlAllowOrigin, HeaderNames.AccessControlAllowMethods };
```

Comment:
...other part of the styling.

```suggestion
            HeaderNames.AccessControlAllowOrigin, HeaderNames.AccessControlAllowMethods
        };
```